xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -59,10 +59,23 @@ void CGUIWindowPVRGuide::OnInitWindow()
 
   CGUIEPGGridContainer *epgGridContainer =
     dynamic_cast<CGUIEPGGridContainer*>(GetControl(m_viewControl.GetCurrentControl()));
+
   if (epgGridContainer)
-    epgGridContainer->GoToNow();
+  {
+    CPVRChannelGroupPtr group(GetGroup());
+    if (!group)
+      group = g_PVRManager.GetPlayingGroup(m_bRadio);
```

Comment:
meh the call to ` CGUIWindowPVRBase::OnInitWindow()` is done later, could we move the call to the top before this?